src/main.rs

```diff
@@ -294,10 +302,11 @@ async fn get_avl_proof(
             rpc_params![
                 state.bridge_contract_address.as_str(),
                 [B256::from_slice(&result[..]).to_string()],
-                "finalized"
+                block_hash
```

Comment:
WDYT of defaults to `latest` or `finalized` if no `blockHash` is passed on path parms?

e.g:
```
curl https://domain.com/avl/proof/:message_id
```

Reply:
Proof must be fetched for the particular block (slot) that is stored in the vector pallet. If passed the `latest`/`finalized` then the proof will be from the block that is not the one stored in the vector pallet.